CovertInfiltration/Localization/CovertInfiltration.int

```diff
@@ -140,196 +140,184 @@ strOverviewHeader=INFILTRATION
 
 [Activity_CounterDarkEvent X2ActivityTemplate_Infiltration]
-strOverviewDescription=Destroy critical piece of infrastructure
+strOverviewDescription=Destroy ADVENT Infrastructure
 ActionRewardDisplayName=Counter Dark Event
 
 [Activity_SupplyRaid X2ActivityTemplate_Infiltration]
-strOverviewDescription=Raid ADVENT supply convoy
-ActionRewardDisplayName=Enemy materiel
+strOverviewDescription=Raid ADVENT Supply Convoy
+ActionRewardDisplayName=Enemy Materiel
 
 [Activity_CaptureInformant X2ActivityTemplate_Infiltration]
 strOverviewDescription=Neutralize ADVENT VIP
 ActionRewardDisplayName="Enemy Intelligence"
 ActionRewardDetails="Datapad (on Kill or Capture) and Intel Cache (on Capture)"
 
 [Activity_FacilityInformant X2ActivityTemplate_Infiltration]
-strOverviewDescription=Neutralize ADVENT VIP
+strOverviewDescription=Neutralize AVATAR VIP
 ActionRewardDisplayName="Enemy Intelligence"
 ActionRewardDetails="Datapad (on Kill or Capture) and Facility Lead (On Capture)"
 
 [Activity_GatherIntel X2ActivityTemplate_Assault]
-strOverviewDescription=Protect a datatap
-MissionPinLabel=Protect a datatap
+strOverviewDescription=Protect Resistance Datatap
+MissionPinLabel=Resistance Datatap
 
 [Activity_GatherSupplies X2ActivityTemplate_Assault]
-strOverviewDescription=Extract supplies
-MissionPinLabel=Extract supplies
+strOverviewDescription=Extract Supplies
+MissionPinLabel=Supplies Extraction
 
 [Activity_LandedUFO X2ActivityTemplate_Assault]
 strOverviewDescription=Secure the UFO
 MissionPinLabel="Landed UFO"
 
 [Activity_PrepareCounterDE X2ActivityTemplate_CovertAction]
-strOverviewDescription=Prepare an operation to counter the Dark Event
+strOverviewDescription=Identify infrastructure supporting the Dark Event
 
 [Activity_PrepareFactionJB X2ActivityTemplate_CovertAction]
-strOverviewDescription=Plan the jailbreak of a faction soldier
+strOverviewDescription=Consult the faction about a jailbreak of their solder
 
 [Activity_PrepareFacility X2ActivityTemplate_CovertAction]
-strOverviewDescription=Investigate the possibility of a raid on AVATAR project facility
+strOverviewDescription=Pinpoint collaborator assisting AVATAR facility construction
 
 [Activity_PrepareUFO X2ActivityTemplate_CovertAction]
-strOverviewDescription=Prepare the raid on an UFO
+strOverviewDescription=Sabotage the landed UFO's takeoff systems
 
 [Activity_WaitDE X2ActivityTemplate]
-strOverviewHeader=WAIT
-strOverviewDescription=Our friends in the resistance caught wind of ADVENT's project. They are working on finding a way to counter it
+strOverviewHeader="WAIT"
+strOverviewDescription="Our friends in the resistance have caught wind of an ADVENT project and are investigating solutions"
 
 [ActivityChain_CounterDarkEvent X2ActivityChainTemplate]
-strTitle=Counter Dark Event
-strDescription=Placeholder description
+strTitle="Counter Dark Event"
+strDescription="Smash infrastructure critical to ADVENT's counteroperations against XCOM."

```

Comment:
Pretty sure `strDescription` isn't used on `ActivityChain_CounterDarkEvent` - instead the delegates picks between the hidden and the named DE description